coderd/database/migrations/000209_custom_roles.up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE custom_roles (
+	-- name is globally unique. Org scoped roles have their orgid appended
+	-- like:  "name":"organization-admin:bbe8c156-c61e-4d36-b91e-697c6b1477e8"
+	name text primary key,
```

Comment:
What's the benefit of squashing the OrgID into the name?
Will this not make it more difficult if we want to filter roles by org?
Why not have it as a separate column?

Reply:
I was thinking about this, and I'm not 100% sure at present. Maybe another column is better. 

This PR only addresses site wide permissions, and once multi-org is a bit farther, I will add org permissions. When I get to that step 2, I feel I'll be able to make a better judgement call.

Regardless, the name will likely include the org id even if there is an org column. Because that is the name of the role. Even our builtins have the org appended, and the ui just ignores it.